libs/deal-ticket/src/components/market-info/info-market.tsx

```diff
@@ -254,6 +270,32 @@ export const Info = ({ market }: InfoProps) => {
         />
       ),
     },
+    {
+      title: t('Liquidity'),
+      content: (
+        <MarketInfoTable
+          data={{
+            targetStake: market.data && market.data.targetStake,
+            suppliedStake: market.data && market.data?.suppliedStake,
+            marketValueProxy: market.data && market.data.marketValueProxy,
+          }}
+          decimalPlaces={
+            market.tradableInstrument.instrument.product.settlementAsset
+              .decimals
+          }
+          assetSymbol={assetSymbol}
+          link={
+            <Link
+              target="_blank"
+              className="text-ui dark:text-white text-black underline hover:underline hover:text-black-60 dark:hover:text-white-80"
```

Comment:
chore (blocking): None of these classes are required, and some are from the old config. Lets just use the raw Link component styles which should handle underline styles

Reply:
yeah, happy to remove them now. updated